modules/tensor_mechanics/src/materials/ADLAROMANCEStressUpdateBase.C

```diff
@@ -192,29 +226,25 @@ ADLAROMCreepStressUpdate<compute_stage>::computeResidual(const ADReal & effectiv
 
   if (_verbose && compute_stage == RESIDUAL)
   {
-    Moose::out.precision(9);
-    Moose::out << std::scientific;
-    Moose::out << "Need to see what is going into the rom model: \n";
+    // Moose::out.precision(9);
```

Comment:
I'd prefer to not leave commented out code like this in the file. We either want it there or we don't.

Reply:
Whoops! Left over.